ChangeLog

```diff
@@ -17,7 +17,9 @@ What's New in astroid 2.12.9?
 =============================
 Release date: TBA
 
+* Fixed a crash on ``namedtuples`` that use ``typename`` to specify their name.
 
```

Comment:
```suggestion


```
Keeping the number of line will prevent conflict later on because the diff will be contiguous.